Calculator/src/main/java/com/microsoft/device/display/samples/calculator/ui/pages/components/Answer.kt

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.device.display.samples.calculator.ui.pages.components
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import com.microsoft.device.display.samples.calculator.models.equationStringEquivalent
+import com.microsoft.device.display.samples.calculator.ui.pages.calculatorModel
+
+@Composable
+fun Answer(modifier: Modifier = Modifier) {
+    val eq: String = if (!calculatorModel.isOnX) " " + equationStringEquivalent(calculatorModel.currentEquation) + " " else ""
+    val y: String = if (!calculatorModel.isOnX) calculatorModel.y else ""
+    LazyColumn(
+        modifier = modifier
+            .padding(
+                start = 20.dp,
+                top = 0.dp,
+                end = 20.dp,
+                bottom = 0.dp
```

Comment:
Since the default value for these padding values is already 0.dp, you can delete the top/bottom parameters
```suggestion
                start = 20.dp,
                end = 20.dp,
```